packages/components/src/sandbox/index.js

```diff
@@ -193,7 +193,7 @@ class Sandbox extends Component {
 			<FocusableIframe
 				iframeRef={ this.iframe }
 				title={ title }
-				scrolling="no"
+				scrolling="no" /* TODO: Remove */
```

Comment:
Context?

Reply:
I left it in because I was worried I'd forget to remove it and if I pushed this it would alert me to do it, haha. So I have to do that 😆